lib/tasks/argo.rake

```diff
@@ -79,6 +97,23 @@ namespace :argo do
     desc "Overwrite Solr configs and JARs"
     task :config => %w[argo:solr:config] do   # TODO: argo:fedora:config
     end
+
+    # Note: this rake task is failing, with error:
+    # Errno::ECONNREFUSED: Connection refused - connect(2) for "localhost" port 8983
+    #
+    # desc "Start Jetty with Fedora/Solr checks"
+    # task :start do
+    #   Rake::Task['jetty:restart'].invoke
+    #   fedora_url = 'http://localhost:8983/fedora/'
+    #   puts "Checking Solr"
+    #   if web_response_ok? 'http://localhost:8983/solr/'
+    #     puts "Success! Solr responds"
+    #   end
+    #   if web_response_ok? 'http://localhost:8983/fedora/'
+    #     puts "Success! Fedora responds."
+    #   end
+    # end
+
```

Comment:
There seemed to be some consensus that rake tasks could startup jetty and check the Solr/Fedora instances, so this was an attempt to do that.  For whatever reason, it's not working.  Uncomment the code and see if you can get it working because I can't seem to get past some problems with webmock -- I hope so because it could save a lot of time required to hit the browser refresh button when you startup jetty.  Using the configured values is a really good idea and an excellent reason to do this in rake rather than a shell script (although in a 'development' env the URLs are usually the hydra-jetty default ports etc).  (In any case, line 107 should have been deleted anyway.)